### PR TITLE
feature: 新增数据查询节点

### PR DESCRIPTION
**Pull request by neronkl**

### CODE REVIEW

File: itsm/workflow/handler/worksheet_fields_handler.py [hunk 1]
```diff
@@ -118,6 +120,71 @@ def copy_fields_from_worksheet_field(self, worksheet_field_ids, state, service):
                     logger.info("新增流程字段id：{field}".format(field=field.id))
                     field_ids.append(field.id)
 
+        return field_ids
+
+    def _create_global_field(self, data_struct, state):
```
Comment (hanshuaikang) [inline itsm/workflow/handler/worksheet_fields_handler.py:125]: 注释多写一些

File: itsm/workflow/handler/worksheet_fields_handler.py [hunk 2]
```diff
@@ -118,6 +120,71 @@ def copy_fields_from_worksheet_field(self, worksheet_field_ids, state, service):
...
+                    "type": item["type"],
+                    "state_id": state.id,
+                    "flow_id": state.workflow_id,
+                    "meta": item["meta"],
+                }
+                fields_object = GlobalVariable.objects.filter(
+                    state_id=state.id, is_deleted=False
+                ).values("id", "meta")
+                exist_flag = None
+                for field in fields_object:
```
Comment (hanshuaikang) [inline itsm/workflow/handler/worksheet_fields_handler.py:167]: if 嵌套太深了

File: nocode/worksheet/handlers/worksheet_field_handler.py [hunk 3]
```diff
@@ -149,7 +151,23 @@ def update_data(self, item, context):
...
+        if auto_number_fields.exists():
+            # 清除定时任务
+            can_reset_items = [
+                item
+                for item in auto_number_fields
+                if not item.meta["config"][0]["period_type"].isdigit()
+            ]
+            for item in can_reset_items:
+                key = "{}_{}_{}".format(item.worksheet_id, item.id, item.key)
+                PeriodicTask.objects.filter(
```
Comment (hanshuaikang) [inline nocode/worksheet/handlers/worksheet_field_handler.py:165]: 对应的celery定时任务没有删掉

**Comment by tencentblueking-adm** [comment]
[CLA assistant check](https://bk-cla.bktencent.com:443/TencentBlueKing/bk-nocode?pullRequest=115) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://bk-cla.bktencent.com:443/TencentBlueKing/bk-nocode?pullRequest=115) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://bk-cla.bktencent.com:443/check/TencentBlueKing/bk-nocode?pullRequest=115) it.</sub>

**Pull request closed without merging**